backend/src/features/game-identifier/game.identifier.spec.js

```diff
@@ -791,14 +793,149 @@ describe("game.identifier.js", function () {
   });
 });
 
+describe(".updateGamesWithoutDetails.", function () {
+  describe("Finds no missing game properties in the database and stops", function () {
+    beforeEach(async function () {
+      this.options = {
+        batchSize: 1,
+      };
+
+      this.steamClientMock = createSteamMock([]);
+      this.steamAppsRepository = createSteamAppsRepositoryMock();
+      this.gamesRepository = createGamesRepositoryMock([]);
+      this.historyChecksRepository = createHistoryChecksRepositoryMock();
+
+      this.identifier = new GameIdentifier(
+        this.steamClientMock,
+        this.steamAppsRepository,
+        this.gamesRepository,
+        this.historyChecksRepository,
+        createLoggerMock(),
+        this.options,
+      );
+
+      await this.identifier.updateGamesWithoutDetails();
+    });
+
+    it("getGamesWithoutDetails was called once", function () {
+      expect(this.gamesRepository.getGamesWithoutDetails).toHaveBeenCalledTimes(1);
+    });
+
+    it("getGamesWithoutDetails was called with 'this.options.batchSize'", function () {
+      expect(this.gamesRepository.getGamesWithoutDetails).toHaveBeenCalledWith(
+        this.options.batchSize,
+      );
+    });
+
+    it("getSteamDbHtmlDetailsPage was not called", function () {
+      expect(this.steamClientMock.getSteamDbHtmlDetailsPage).toHaveBeenCalledTimes(0);
+    });
+
+    it("updateGameDetails was not called", function () {
+      expect(this.gamesRepository.updateGameDetails).toHaveBeenCalledTimes(0);
+    });
+  });
+
+  describe("Finds two games with missing properties", function () {
+    beforeEach(async function () {
+      this.options = {
+        batchSize: 2,
+        unitDelay: 0,
+      };
+
+      const releaseDate = "";
```

Comment:
I would extract this line till line 866 into a function fon readability. But its not mandatory!